types/fg-loadcss/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for fg-loadcss 3.1
+// Project: https://github.com/filamentgroup/loadCSS
+// Definitions by: Noah Overcash <https://github.com/ncovercash>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/fg-loadcss/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me